hack/hypershift-package/hypershift-artifacts-template.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: selectorsyncset-template
+
+parameters:
+- name: IMAGE_TAG
+  required: true
+- name: REPO_NAME
+  value: route-monitor-operator
+  required: true
+- name: PACKAGE_IMAGE
+  required: true
+- name: PACKAGE_TAG
+  required: true
+
+objects:
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: route-monitor-operator
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+    resourceApplyMode: Sync
+    resources:
+    - apiVersion: policy.open-cluster-management.io/v1
+      kind: Policy
+      metadata:
+          annotations:
+              policy.open-cluster-management.io/categories: CM Configuration Management
+              policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+              policy.open-cluster-management.io/standards: NIST SP 800-53
+          name: hs-mgmt-route-monitor-operator
+          namespace: openshift-acm-policies
+      spec:
+          disabled: false
+          policy-templates:
+              - objectDefinition:
+                  apiVersion: policy.open-cluster-management.io/v1
+                  kind: ConfigurationPolicy
+                  metadata:
+                      name: hs-mgmt-route-monitor-operator
+                  spec:
+                      evaluationInterval:
+                          compliant: 2h
```

Comment:
Should this be 2min (unsure about syntax), maybe 2m based on https://issues.redhat.com/browse/OSD-17493?

Reply:
Fixed